### DISCUSSION TITLE
darkmode support

### DISCUSSION THREAD

**Discussion by devrob20** [IceWhaleTech/CasaOS#248] [Ideas]
for ppl with low vision darkmode is often a must-have , there is an official way to make html and css work with the darkmode setting of browsers and operating systems.

please implement it.
thanks.

https://css-tricks.com/a-complete-guide-to-dark-mode-on-the-web/

**Comment by Lauren-ED209** [comment]
I've tranfered this request to development schedule for further discussion! Thanks!

**Comment by alekssze** [comment]
I also would like Dark Mode With Ability to change Background

**Comment by CMSprocket** [comment]
Hi, I'm a new CasaOS user and I am looking to see if there is any update on this.

Dark mode would be very beneficial on at least the App Store and Files Pages. Something either toggleable or a system/browser check would be very nice. Thank you!

P.S. I'm loving using it by the way.
